.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+# Copyright (C) Nitrokey GmbH
+# SPDX-License-Identifier: CC0-1.0
+
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@master
+    - name: Install Python dependencies
+      run: |
+        python -m pip install toml
```

Comment:
Maybe the commands should be defined in a more widespread format?

Reply:
I think toml is fine, especially for a Rust crate. :)  Python 3.11 also brings tomllib.